### PR TITLE
[Smart Start] Static contact form and thank you page

### LINKED ISSUE #8257: [Smart start] static contact us form
The static contact page can be copied from the maas.io/contact-us page.
This form will be in the smart-start/contact-us page

### PR DESCRIPTION
**Pull request by carkod**
## Done

- Add static contact us form and thank-you page

## QA

- Go to demo https://ubuntu-com-8334.demos.haus/smart-start/contact-us
- Fill out the form

## Issue / Card

Fixes #8257

### CODE REVIEW

File: templates/smart-start/base_smart-start.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% extends "templates/base.html" %}
+
+{% block outer_content %}
+  {% block content %}{% endblock %}
+
+  <!-- Set default Marketo information for contact form below-->
+  <div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/smart-start" data-form-id="3591" data-lp-id="" data-return-url="/support/thank-you" data-lp-url="/support/contact-us">
```
Comment (pmahnke) [inline templates/smart-start/base_smart-start.html:7]: the data-return-url should be `/smart-start/thank-you`

Comment (pmahnke) [reply templates/smart-start/base_smart-start.html:7]: the data-lp-url should be `/smart-start/contact-us`

File: templates/smart-start/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+{% block title %}Contact us | SMART START{% endblock %}
+
+{% block content %}
+
+{% with h1='Contact us about SMART START',
+  intro_text="Get your IoT project started with SMART START",
+  formid="3591",
+  lpId="",
+  side1="shared/forms/_desktop_help.html",
```
Comment (pmahnke) [inline templates/smart-start/contact-us.html:11]: I don't think you want either `side1` or `side2`

File: templates/smart-start/contact-us.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{% extends "smart-start/base_smart-start.html" %}
+
+{% block title %}Contact us | SMART START{% endblock %}
+
+{% block content %}
+
+{% with h1='Contact us about SMART START',
+  intro_text="Get your IoT project started with SMART START",
+  formid="3591",
```
Comment (pmahnke) [inline templates/smart-start/contact-us.html:9]: did we get this id from Lucia?

**Review by pmahnke — CHANGES_REQUESTED** [review]
A few comments inline - also, does this have your other guide move changes?

File: templates/smart-start/contact-us.html (hunk 3 above)
Comment (carkod) [reply templates/smart-start/contact-us.html:9]: Yes, but if you could double check, I got it from (her comment)[https://docs.google.com/document/d/1jjNxnttbJfV-jtMqMIooyA6saba4rU_mB4UTj_uv7R0/edit?disco=AAAAJWIqo14]

Comment (carkod) [reply templates/smart-start/contact-us.html:9]: Also need somebody to proofread those headings and intro_text as I made it up, maybe we can say something more meaningful

File: templates/smart-start/contact-us.html (hunk 2 above)
Comment (carkod) [reply templates/smart-start/contact-us.html]: Done

File: templates/smart-start/base_smart-start.html (hunk 1 above)
Comment (carkod) [reply templates/smart-start/base_smart-start.html]: Thanks. Done

**Comment by carkod** [comment]
> A few comments inline - also, does this have your other guide move changes?

Just rebased. So yes.

File: templates/smart-start/contact-us.html (hunk 3 above)
Comment (carkod) [reply templates/smart-start/contact-us.html:9]: Done

**Review by pmahnke — APPROVED** [review]
Seems ok, but I didn't get the form through, but I think that isn't related to this PR

**Comment by carkod** [comment]
> Seems ok, but I didn't get the form through, but I think that isn't related to this PR

So you are not seeing the thank-you page after you submit the form? I just tried and it works

**Comment by pmahnke** [comment]
I do; however, in the network tab, the save isn't returning a 200

**Pull request merged**